README.md

```diff
@@ -17,10 +17,13 @@ swap(first_arr, second_array)
 console.log(first_arr) // [4, 5, 6]
 console.log(second_array) // [1, 2, 3]
 ```
-
+Think I blew it :/
 ```js
 function swap(arr1, arr2) {
-  // -- YOUR ANSWER HERE --
+  arr1 = arr2
+  arr2 = arr1
+  console.log(arr1)
+  console.log(arr2)
```

Comment:
Did you try running this code? It is a good idea but you lose access to arr1 before you finish the swap.